lib/AST/DeclContext.cpp

```diff
@@ -518,7 +518,10 @@ ResilienceExpansion DeclContext::getResilienceExpansion() const {
       // we serialize the parent's body.
       if (AFD->getDeclContext()->isLocalContext())
         continue;
-      
+
+      if (AFD->isInvalid())
```

Comment:
What is the actual problem here -- the context doesn't have accessibility? It seems like this fix is just masking the bug.

I'd like us to move away from checks like isInvalid() and instead query the correct invariant, like hasAccessibility(), or maybe just fix the underlying issue where the accessibility wasn't set correctly.